### PR TITLE
zh-CN: cr2 Day 1 translations

### PR DESCRIPTION
**Pull request by henrif75**
#324 #1463
*Stats*
Before: 1018 translated messages, 368 fuzzy translations, 2074 untranslated messages.
After: 1250 translated messages, 530 fuzzy translations, 1680 untranslated messages.

You can ignore _fuzzy_ entries in sections outside Day 1, they should be addressed by other PRs.

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -247,12 +253,12 @@ msgstr "控制流"
...
 #: src/SUMMARY.md:74 src/pattern-matching/exercise.md:1
 msgid "Exercise: Expression Evaluation"
-msgstr ""
+msgstr "练习：表达式求值"
 
 #: src/SUMMARY.md:76 src/methods-and-traits.md:1
 #, fuzzy
 msgid "Methods and Traits"
-msgstr "读取和写入"
+msgstr "审查和审核"
```
Comment (emmali01) [inline po/zh-CN.po:261]:
```suggestion
msgstr "方法和特征"
```

File: po/zh-CN.po [hunk 2]
```diff
@@ -343,7 +351,7 @@ msgstr "`HashMap`"
 #: src/SUMMARY.md:102 src/std-types/exercise.md:1
 #, fuzzy
 msgid "Exercise: Counter"
-msgstr "习题"
+msgstr "练习：几何图形"
```
Comment (emmali01) [inline po/zh-CN.po:354]:
```suggestion
msgstr "练习：计数器"
```

File: po/zh-CN.po [hunk 3]
```diff
@@ -383,7 +391,7 @@ msgstr "闭包"
 #: src/SUMMARY.md:112 src/std-traits/exercise.md:1
 #, fuzzy
 msgid "Exercise: ROT13"
-msgstr "习题"
+msgstr "练习：二进制文件树"
```
Comment (emmali01) [inline po/zh-CN.po:394]:
```suggestion
msgstr "练习：ROT13"
```

File: po/zh-CN.po [hunk 4]
```diff
@@ -472,7 +482,7 @@ msgstr "互操作性"
 #: src/SUMMARY.md:143 src/borrowing/exercise.md:1
 #, fuzzy
 msgid "Exercise: Health Statistics"
-msgstr "健康统计"
+msgstr "练习：电梯事件"
```
Comment (emmali01) [inline po/zh-CN.po:485]:
```suggestion
msgstr "练习：健康统计"
```

File: po/zh-CN.po [hunk 5]
```diff
@@ -662,7 +676,7 @@ msgstr "实现 Unsafe Trait"
 #: src/SUMMARY.md:200
 #, fuzzy
 msgid "Exercise: FFI Wrapper"
-msgstr "安全 FFI 封装容器"
+msgstr "练习：二进制文件树"
```
Comment (emmali01) [inline po/zh-CN.po:679]:
```suggestion
msgstr "练习：FFI 封装容器"
```

File: po/zh-CN.po [hunk 6]
```diff
@@ -747,24 +761,27 @@ msgstr "测试模块"
 #: src/SUMMARY.md:226
 #, fuzzy
 msgid "Rust Bridge"
-msgstr "Rust 二进制文件"
+msgstr "C++ 桥接"
```
Comment (emmali01) [inline po/zh-CN.po:764]:
```suggestion
msgstr "Rust 桥接"
```

File: po/zh-CN.po [hunk 7]
```diff
@@ -747,24 +761,27 @@ msgstr "测试模块"
...
+#, fuzzy
 msgid "C++ Bridge"
-msgstr ""
+msgstr "C++ 桥接"
 
 #: src/SUMMARY.md:229 src/android/interoperability/cpp/shared-types.md:1
 #, fuzzy
 msgid "Shared Types"
-msgstr "标量类型"
+msgstr "共享枚举"
```
Comment (emmali01) [inline po/zh-CN.po:779]:
```suggestion
msgstr "共享类型"
```

File: po/zh-CN.po [hunk 8]
```diff
@@ -778,19 +795,22 @@ msgstr "错误处理"
...
+#, fuzzy
 msgid "Building for Android: C++"
-msgstr ""
+msgstr "针对 Android 进行构建：C++"
 
 #: src/SUMMARY.md:235
+#, fuzzy
 msgid "Building for Android: Genrules"
-msgstr ""
+msgstr "针对 Android 进行构建：genrule"
```
Comment (emmali01) [inline po/zh-CN.po:808]:
```suggestion
msgstr "针对 Android 进行构建：Genrule"
```

File: po/zh-CN.po [hunk 9]
```diff
@@ -806,35 +826,39 @@ msgid "Exercises"
...
 msgid "Visual Studio Code"
-msgstr ""
+msgstr "Visual Studio Code"
 
 #: src/SUMMARY.md:252 src/SUMMARY.md:257 src/SUMMARY.md:265 src/SUMMARY.md:278
 #: src/exercises/chromium/third-party.md:1
 #, fuzzy
 msgid "Exercise"
-msgstr "习题"
+msgstr "练习：几何图形"
```
Comment (emmali01) [inline po/zh-CN.po:861]:
```suggestion
msgstr "练习"
```

File: po/zh-CN.po [hunk 10]
```diff
@@ -881,89 +905,102 @@ msgid "Error Handling: PNG Example"
...
+#, fuzzy
 msgid "Bringing It Together - Exercise"
-msgstr ""
+msgstr "综合应用练习"
 
 #: src/SUMMARY.md:280 src/exercises/chromium/solutions.md:1
 #, fuzzy
 msgid "Exercise Solutions"
-msgstr "解答"
+msgstr "练习：表达式求值"
```
Comment (emmali01) [inline po/zh-CN.po:987]:
```suggestion
msgstr "练习题解答"
```

File: po/zh-CN.po [hunk 11]
```diff
@@ -1009,8 +1046,9 @@ msgstr "其他项目"
 
 #: src/SUMMARY.md:301 src/exercises/bare-metal/compass.md:1
 #: src/exercises/bare-metal/solutions-morning.md:3
+#, fuzzy
 msgid "Compass"
-msgstr "罗盘"
+msgstr "对比项"
```
Comment (emmali01) [inline po/zh-CN.po:1051]:
```suggestion
msgstr "罗盘"
```

File: po/zh-CN.po [hunk 12]
```diff
@@ -3214,10 +3316,13 @@ msgstr ""
 "`return`。"
 
 #: src/control-flow-basics/blocks-and-scopes.md:49
+#, fuzzy
 msgid ""
 "Show that a variable's scope is limited by adding a `b` in the inner block "
 "in the last example, and then trying to access it outside that block."
 msgstr ""
+"通过在上一个示例的内部代码块中添加 ab\\`，并尝试在该代码块之外访问它，表明变"
```
Comment (emmali01) [inline po/zh-CN.po:3324]:
```suggestion
"通过在上一个示例的内部代码块中添加 `b`，并尝试在该代码块之外访问它，表明变"
```

**Review by emmali01 — APPROVED** [review]

**Review by SketchK — APPROVED** [review]

**Comment by henrif75** [comment]
Thanks @emmali01 @SketchK !

**Pull request merged**